### PR TITLE
add CLI to runs

### LINKED ISSUE #83: Make `num_trials` a CLI parameter for runs
With a soft default for local development, but the parameter allows us to easily run this at larger scales on the clusters

### PR DESCRIPTION
**Pull request by Opeyem1a**
Goals:
- Solving #83 

typer is a library I added to skip having to do CLI parameter validation + making the `--help` command, seems nifty

closes #83

### CODE REVIEW

**Comment by Opeyem1a** [comment]
Current dependencies on/for this PR:
* main
  * **PR #82** <a href="https://app.graphite.dev/github/pr/Teamable-Analytics/algorithms/82" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Teamable-Analytics/algorithms/82?utm_source=stack-comment).

File: benchmarking/runs/concentrate_many_attributes.py [hunk 1]
```diff
@@ -17,14 +17,13 @@
 from models.enums import ScenarioAttribute, Gender, Race
 
 
-def concentrate_many_attributes():
+def concentrate_many_attributes(num_trials: int = 10):
```
Comment (Hedgemon4) [inline benchmarking/runs/concentrate_many_attributes.py:20]: Do you not need to surround the function call here with `typer.run(concentrate_many_attributes)` as you did in the previous file?

**Review by Hedgemon4 — CHANGES_REQUESTED** [review]

**Review by Hedgemon4 — APPROVED** [review]
LGTM

**Review by justino599 — APPROVED** [review]
lgtm

**Pull request merged**